soc/arm/nxp_imx/rt/soc_rt10xx.c

```diff
@@ -224,11 +224,16 @@ static ALWAYS_INLINE void clock_init(void)
 	defined(CONFIG_MEMC_MCUX_FLEXSPI) && \
 	DT_NODE_HAS_STATUS(DT_NODELABEL(flexspi), okay)
 	CLOCK_DisableClock(kCLOCK_FlexSpi);
-	CLOCK_InitUsb1Pfd(kCLOCK_Pfd0, 24);
+#endif
+
+#if DT_NODE_HAS_STATUS(DT_NODELABEL(flexspi), okay) && \
+	defined(CONFIG_MEMC_MCUX_FLEXSPI)
```

Comment:
Why is the separated out from the previous conditional macrp

Reply:
The conditional is wrong so the clock was never setup. Atleast that is how I saw it on the board.